app/sprinkles/account/src/Authenticate/Authenticator.php

```diff
@@ -122,6 +122,11 @@ public function __construct(ClassMapper $classMapper, Session $session, Config $
             $this->rememberMe->getCookie()->setExpireTime($this->config['remember_me.expire_time']);
         }
 
+        // Set domain, if specified
+        if ($this->config->has('remember_me.domain') && ($this->config->has('remember_me.domain') != null)) {
```

Comment:
Simple note, you have extra `()` pair in there : 

```
if ($this->config->has('remember_me.domain') && $this->config->has('remember_me.domain') != null) {
```

Reply:
I copied the coding style of the line above it.